apps/condo/migrations/20220209123400_0099_manual_set_ticket_address_info.js

```diff
@@ -0,0 +1,16 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+            UPDATE "Ticket" AS t
+            SET "propertyAddress" = p.address,
+                "propertyAddressMeta" = p."addressMeta"
+            FROM "Property" AS p
```

Comment:
I'm expecting something like `join` here. Is this query really work well? Did you test it?

Reply:
Tested it locally in datagrip

Reply:
https://stackoverflow.com/questions/7869592/how-to-do-an-update-join-in-postgresql